en-us/PowerGRR-help.xml

```diff
@@ -3233,7 +3233,7 @@ False</dev:code>
       <command:example>
         <maml:title>-------------------------- Example 1 --------------------------</maml:title>
         <dev:code>PS C:\&gt; $res = Get-GRRHuntResult -HuntId H:AAAAAAAA -Credential $creds
-PS C:\&gt; ($res.items.payload.stat_entry.aff4path).substring(31) | sort -u
+PS C:\&gt; ($res.items.payload.stat_entry.pathspec.path).substring(31) | sort -u
```

Comment:
substring(31) is not needed anymore because we filtered out the aff4 prefix using the substring function.

```suggestion
PS C:\&gt; $res.items.payload.stat_entry.pathspec.path | sort -u
```